cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/AbstractAssemblyOperator.java

```diff
@@ -260,13 +260,14 @@ public final CountDownLatch reconcileAll(String trigger, String namespace, Label
         log.debug("reconcileAll({}, {}): desired resources with labels {}: {}", assemblyType, trigger, selector, desiredNames);
 
         // get resources with kind=cluster&type=kafka (or connect, or connect-s2i)
-        List<? extends HasMetadata> resources = getResources(namespace);
+        Labels resourceSelector = Labels.EMPTY.withKind(assemblyType.name);
+        List<? extends HasMetadata> resources = getResources(namespace, resourceSelector);
         // now extract the cluster name from those
         Set<String> resourceNames = resources.stream()
                 .filter(r -> !r.getKind().equals(kind)) // exclude desired resource
                 .map(Labels::cluster)
                 .collect(Collectors.toSet());
-        log.debug("reconcileAll({}, {}): Other resources with labels {}: {}", assemblyType, trigger, selector, resourceNames);
```

Comment:
Do we no longer use the `selector` parameter? We should remove it if we don't use it.

Reply:
@tombentley We use all the selectors. But the one passed from `ClusterController` is always `Labels.EMPTY`, since the CRs do not have any mandatory labels anymore. We can remove that one if you want.

Reply:
I think that would make sense.